reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<blockquote>to allow the RL agent interact with the environment to collect rewards.</blockquote><p>to <strong>let</strong> the RL agent interact with the environment <strong>and</strong> collect rewards.</p><p> </p><blockquote>Checkpointed data from the previously trained models is passed on for evaluation / inference in the checkpoint channel.</blockquote><p>I didn't understand what checkpointed data and previously trained models meant. How about: <strong>Download the trained model from the batch RL job. Pass the model in the checkpoint channel for the evaluation job.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
We are using checkpoint for evaluation but not the trained model, although they're very closed to each other. I'll just keep **Checkpointed data is passed on for evaluation / inference in the checkpoint channel.**